client/src/CommentBox.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import $ from 'jquery';
+import CommentForm from './AddCommentForm';
```

Comment:
It's possible that you can import the default exported value as whatever name you want.